CONTRIBUTING.md

```diff
@@ -0,0 +1,93 @@
+# Contributing (Code) to Python Community News
+
+Thank you for taking interest in contributing to this project.
+
+
+## Before you start
+This project has a [Code of Conduct](CODE_OF_CONDUCT.md) that you MUST follow in order to participate in this project in any way.
+
+This document is intended to help first-time or returning contributors to the project. It covers making contributions outside of the scheduled content and topics. For more information on how to submit a topic or conference, please see the [README](README.md).
+
+## About the code in this project
+
+You can break the code into essentially four parts:
+- [Contributing (Code) to Python Community News](#contributing-code-to-python-community-news)
+  - [Before you start](#before-you-start)
+  - [About the code in this project](#about-the-code-in-this-project)
+    - [For All Types of Contributions](#for-all-types-of-contributions)
+    - [Automation with Github via GH Actions](#automation-with-github-via-gh-actions)
+  - [General Administration around GitHub](#general-administration-around-github)
+    - [Deployting to our third-party platforms via API](#deployting-to-our-third-party-platforms-via-api)
+  - [Deploying of content to the web via Render Engine](#deploying-of-content-to-the-web-via-render-engine)
+
+You are able to contribute to all four of these parts but in different ways.
+
+### For All Types of Contributions
+To ensure that everyone is one the same page, please follow this process for all types of contributions.
+
+1. Read and follow [Code of Conduct](CODE_OF_CONDUCT.md)
+2. Create an issue on GitHub with the title of your perspective change. Use the approriate labels for the type of contribution you are making. Examples of appropriate labels are:
+    - `bug`
+    - `feature`
+    - `documentation`
+    - `gh action`
+
+    For bugs and features include a label for the area of the project that is affected. Examples of appropriate labels are:
+        - `newsletter`
+        - `website`
+        - `podcast`
+
+3. Specify that you will work on this issue. This lets others know that someone is actively working on the project.
+4. Fork the repository on your profile and make changes locally. 
```

Comment:
This section feels a bit incomplete. I like the section as a whole, but forking the repository is closer to the beginning of getting a change through than it is to the end.

Reply:
It is, sadly I'm wasn't sure what the steps would be after that. With a little more clarity I hope to add some of the steps as good first issues for others as we begin to better define them.